Apps/WebClient/src/ClientApp/src/models/dateWrapper.ts

```diff
@@ -43,6 +43,11 @@ export class DateWrapper {
      */
     private readonly _internal_date: DateTime;
 
+    /**
+     * The 'yyyy-MM-dd' tokens to format the string as.
+     */
+    public static yyyy_MM_dd_format = "yyyy-MM-dd";
```

Comment:
If we all like this approach to centralize the yyyy_MM-format, I can create a task so that we can refactor code to use this instead of having the following code in many places:
`private formatDate(date: string): string {
        return new DateWrapper(date).format("yyyy-MM-dd");
    }`

Reply:
Yes, we should create a Technical story and have this as a common utility similar to server side Common.Utils.DateTimeFormatter

Reply:
I like the format method, I would rename it formatDateString. having the static string looks quite ugly, however. If we want to have the formats more readable we can have a file about date formats

Reply:
nice comments. code updated